packages/story-editor/src/utils/nativeCopyPasteExpected.ts

```diff
@@ -18,24 +18,31 @@
  * Checks if the native copy paste should proceed,
  * either if text is selected or if there's focus on a text/number input.
  *
- * @return {boolean} If native handling is expected.
+ * @return If native handling is expected.
  */
 function nativeCopyPasteExpected() {
   const { activeElement } = document;
-  const { tagName, type, contentEditable } = activeElement;
 
-  // If it's an input in focus, do native handling.
-  if (
-    'true' === contentEditable ||
-    'textarea' === tagName.toLowerCase() ||
-    ('input' === tagName.toLowerCase() &&
-      ['text', 'number', 'search', 'email', 'tel', 'url'].includes(type))
-  ) {
-    return true;
-  }
+  if (activeElement) {
+    const { tagName, type, contentEditable } = activeElement as
+      | HTMLInputElement
+      | HTMLAnchorElement
```

Comment:
Why anchor elements? `contentEditable` can be applied to any element.

Reply:
Recommended by @miina

Reply:
@spacedmonkey The idea was that previously you had created a custom interface to support `tagName, type, contentEditable`, however, [I mentioned there are existing default types](https://github.com/GoogleForCreators/web-stories-wp/pull/12737#discussion_r1039984914) already. I did not look into it thoroughly though, so there might be some missing.

@barklund I don't think that `contentEditable` and `type` both exists on `Element` / `HTMLElement` so this needs more precision. In this particular case, we are trying to detect if native copy-paste should take over or not.

Reply:
The typing is hard here. It can be multiple types, that is valid. Not all types have the same props, some have contentEditable and other have type. Not sure what to do here other than what as been done. Maybe a custom type.